db/seeds/data/libraries.yml

```diff
@@ -86,7 +86,7 @@
 
     BotBlockAPI api = new BotBlockAPI.Builder()
         .addAuthToken("bots.ondiscord.xyz", "dsag38_auth_token_fda6gs") // Add a supported site and its token
```

Comment:
Does this not get an enum?

Reply:
I wanted to show that it would work with String and an enum respectively...

Reply:
Ah cool.